src/Sulu/Bundle/AdminBundle/Resources/public/scss/modules/layout.scss

```diff
@@ -33,19 +33,17 @@
     }
 
     .page {
-        height: 100%;
+        height: calc(100% - 50px);
```

Comment:
This works quite well, the only bug is if the length of the form is within this 50px range. Then the form cannot be scrolled to its very bottom. This is only possible if the tabs have already disappeared (which happens after scrolling down a bit).